airbyte-integrations/connectors/source-quickbooks/acceptance-test-config.yml

```diff
@@ -0,0 +1,46 @@
+# See [Connector Acceptance Tests](https://docs.airbyte.com/connector-development/testing-connectors/connector-acceptance-tests-reference)
+# for more information about how to configure these tests
+connector_image: airbyte/source-quickbooks:dev
+acceptance_tests:
+  spec:
+    tests:
+      - spec_path: "source_quickbooks/spec.yaml"
+  connection:
+    tests:
+      - config_path: "secrets/config.json"
+        status: "succeed"
+      - config_path: "integration_tests/invalid_config.json"
+        status: "failed"
+  discovery:
+    tests:
+      - config_path: "secrets/config.json"
+  basic_read:
+    tests:
+      - config_path: "secrets/config.json"
+        configured_catalog_path: "integration_tests/configured_catalog.json"
+        empty_streams:
+          - name: vendor_credits
+            bypass_reason: "enable to populate"
```

Comment:
Typo in `bypass_reason`. enable -> unable

Reply:
fixed